spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,45 @@
+feature 'Sign In' do
```

Comment:
All of the tests in this file begin in the same way - feels like a good use case for extracting out into a helper method.